src/video.cpp

```
@@ -65,6 +65,40 @@ using namespace std::literals;
+          previous_capture = config::video.capture;
+          config::video.capture = *capture_override;
+          active = true;
+        }
+      }
+
+      ~scoped_capture_override_t() {
+        if (active) {
+          config::video.capture = previous_capture;
+        }
+      }
+
```

Cppcheck (2.20.0)

[error] Cannot find end of expression

(internalError)